src/System.Windows.Forms/src/System/Windows/Forms/Controls/PropertyGrid/PropertyGridInternal/GridEntry.cs

```
@@ -1478,9 +1505,14 @@ public virtual object[] GetPropertyValueList()
     ///   <see cref="GridEntry"/> needs to be painted.
     ///  </para>
     /// </remarks>
-    public virtual void PaintLabel(Graphics g, Rectangle rect, Rectangle clipRect, bool selected, bool paintFullLabel)
+    public virtual void PaintLabel(
+        Graphics g,
+        Rectangle rect,
+        Rectangle clipRect,
+        bool selected,
+        bool paintFullLabel)
     {
-        PropertyGridView ownerGrid = OwnerGridView;
+        PropertyGridView ownerGrid = OwnerGridView!;
```

Ditto here with adding a comment. Alternatively we can throw `InvalidOperationException` if it is null